python/ray/autoscaler/_private/util.py

```diff
@@ -144,6 +145,13 @@ def merge_setup_commands(config):
     return config
 
 
+def fill_node_type_max_workers(config):
+    """Default per-node max workers to global max_workers."""
```

Comment:
Can you add a comment to explain why global max_workers?
something like: This is equivalent to setting the per node type max workers to infinity with the only constraint being set by the global max_workers.

Reply:
You resolved this comment but it seems like it was not resolved?

Reply:
oh -- my bad, I added something to the docs rather than to the doc string.

Reply:
fixed